src/main/java/tars/Task.java

```diff
@@ -0,0 +1,196 @@
+package tars;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * Class represents a generic class with a Description and isDone status
+ * Serves as a base class for other Task types like ToDos, Deadline, Event
+ *
+ * @author csk
+ * @version 1
+ */
+public class Task {
+    protected String description;
+    protected boolean isDone;
+
+    /**
+     * Constructor for new Task with specified description
+     * The task will be initialised with marked as not done
+     *
+     * @param description (description of Task)
+     */
+    public Task(String description) {
+        this.description = description;
+        this.isDone = false;
+    }
+
+    /**
+     * Returns a status icon depending on the isDone status of the Task
+     *
+     * @return "X" if task is done, " " otherwise returned
+     */
+    public String getStatusIcon() {
```

Comment:
since you used the `this` keyword to refer to the instance of isDone in toher places you should use it here too, for consistency